src/serlio/prtMaterial/PRTMaterialNode.cpp

```diff
@@ -20,6 +20,7 @@
 #include "prtMaterial/PRTMaterialNode.h"
 
 #include "prtMaterial/MaterialInfo.h"
+#include "prtMaterial/MaterialUtils.h"
 
 #include "prtModifier/PRTModifierAction.h"
 

```

Comment:
unused headers:
`prtMaterial/MaterialInfo.h`
`util/MArrayWrapper.h`
`util/Utilities.h`
`serlioPlugin.h`
`prt/StringUtils.h`
`maya/MObject.h`
`<algorithm>`
`<array>`
`<cstdio>`
`<set>`
`<sstream>`

Reply:
done (array and set are needed)

Reply:
make sense 👍